docs/security.html

```diff
@@ -428,12 +428,12 @@ <h3 class="anchor-heading"><a id="security_sasl" class="anchor-link"></a><a href
             <tt>zookeeper.sasl.client.username</tt> to the appropriate name
             (<i>e.g.</i>, <tt>-Dzookeeper.sasl.client.username=zk</tt>).</p>
 
-            <p>Brokers may also configure JAAS using the broker configuration property <code>sasl.jaas.config</code>.
-            The property name must be prefixed with the listener prefix including the SASL mechanism,
-            i.e. <code>listener.name.{listenerName}.{saslMechanism}.sasl.jaas.config</code>. Only one
-            login module may be specified in the config value. If multiple mechanisms are configured on a
-            listener, configs must be provided for each mechanism using the listener and mechanism prefix.
-            For example,
+                    <p>Brokers may also configure JAAS using the broker configuration property <code>sasl.jaas.config</code>.
+                        The property name must be prefixed with the listener prefix including the SASL mechanism,
+                        i.e. <code>listener.name.{listenerName}.{saslMechanism}.sasl.jaas.config</code>. Only one
+                        login module may be specified in the config value. If multiple mechanisms are configured on a
+                        listener, configs must be provided for each mechanism using the listener and mechanism prefix.
+                        For example,
```

Comment:
Why the indentation here? It should be on the same level as the previous paragraph, right?

Reply:
Actually, it seems the above paragraphs should be indented to the right, there are several nested `<ol>` and `<li>` and `<p>` that didn't follow indentation.
I'm fixing these now.

Reply:
I think we misunderstood each other. The line that starts with `The property name ...` should be aligned with `<p>Brokers may also ...`.  I am also fine, if you do not fix that. I think it is understandable and I see that the indentation is not really consistent in this file.